core/lib/eth_client/src/clients/http/query.rs

```diff
@@ -284,7 +285,26 @@ impl EthInterface for QueryClient {
     ) -> Result<Option<Block<H256>>, Error> {
         COUNTERS.call[&(Method::Block, component)].inc();
         let latency = LATENCIES.direct[&Method::Block].start();
-        let block = self.web3.eth().block(block_id).await?;
+        let block = {
+            let include_txs = helpers::serialize(&false);
+
+            let result = match block_id {
+                BlockId::Hash(hash) => {
+                    let hash = helpers::serialize(&hash);
+                    self.web3
+                        .transport()
+                        .execute("eth_getBlockByHash", vec![hash, include_txs])
+                }
+                BlockId::Number(num) => {
+                    let num = helpers::serialize(&num);
+                    self.web3
+                        .transport()
+                        .execute("eth_getBlockByNumber", vec![num, include_txs])
+                }
+            };
+
+            CallFuture::new(result).await?
+        };
```

Comment:
why do you need this change? 
it works as you expected by default

Reply:
It returned `web3::types::Block` which is lacking EIP4844 fields

Reply:
There is a comment explaining why we need copy of `Block`, added for `block` method as well